test/e2e/pivoting_test.go

```diff
@@ -196,9 +197,23 @@ func installIronicBMO(targetCluster framework.ClusterProxy, isIronic, isBMO stri
 	cmd.Dir = path
 	cmd.Env = append(env, os.Environ()...)
 
+	outputPipe, er := cmd.StdoutPipe()
+	Expect(er).To(BeNil(), "Cannot get the stdout from the command")
+	errorPipe, er := cmd.StderrPipe()
+	Expect(er).To(BeNil(), "Cannot get the stderr from the command")
 	err := cmd.Start()
 	Expect(err).To(BeNil(), "Failed to deploy Ironic")
+	data, er := ioutil.ReadAll(outputPipe)
+	Expect(er).To(BeNil(), "Cannot get the stdout from the command")
+	if len(data) > 0 {
+		Logf("Output of the shell: %s\n", string(data))
```

Comment:
maybe:
```suggestion
		Logf("Output of the shell: %v\n", string(data))
```

Reply:
I would use %s in this case because we want to print out a string

Reply:
I asked because you have used %v a bit below for printing _errorData_ string